charts/hivemq-platform/values.schema.json

```diff
@@ -33,6 +33,20 @@
         }
       }
     },
+    "additionalInitContainers": {
```

Comment:
Can we already remove `overrideInitContainers` from the schema? Is this schema used in editors like a `config.xsd`, so customers would directly see that `overrideInitContainers` isn't valid anymore? This could also prevent to guide new customers to only use `additionalInitContainers`.

Reply:
Removed from the schema

Reply:
As long as you can still use the old configuration (if it's valid), I think it should be okay to remove it from the schema. This way new users don't get the old option suggested anymore (at least I assume there are tools using the schema file also for suggestions and auto-completion like an XSD).